events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,356 @@
+	desc = {
+		desc = stasis_trap_outcome.4001.opening
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.4001.owner_discovered
+			}
+			desc = stasis_trap_outcome.4001.owner_secret
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = rage
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		stasis_trap_failure_effect = yes
+	}
+
+	option = { # Abandon scheme
+		name = stasis_trap_outcome.4001.a
+		abandon_scheme_effect = yes
+
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = { # Try again!
+		name = stasis_trap_outcome.4001.b
+		restart_scheme_effect = yes
+
+		ai_chance = {
+			base = 10
+			modifier = {
+				add = 200
+				has_trait = stubborn
+			}
+		}
+	}
+}
+
+########################################
+# TARGET SUCCESS NOTIFICATIONS
+########################################
+
+#I have been stunned!
+stasis_trap_outcome.5001 = {
+	type = character_event
+	title = stasis_trap_outcome.5001.t
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.5001.opening.b
+			}
+			triggered_desc = {
+				trigger = {
+					NOT = {
+						exists = scope:scheme_discovered
+					}
+				}
+				desc = stasis_trap_outcome.5001.opening.a
+			}
+		}
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					scope:stasis_trap_method = flag:trapped_residence
+				}
+				desc = stasis_trap_outcome.5001.trapped_residence
+			}
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:left_portrait
+		triggered_animation = {
+			trigger = {
+				exists = scope:scheme_discovered
+			}
+			animation = scheme
+		}
+		triggered_animation = {
+			trigger = {
+				NOT = { exists = scope:scheme_discovered }
+			}
+			animation = anger
+		}
+	}
+
+	immediate = {
+		show_as_tooltip = {
+			stun_character_effect = {
+				TARGET = scope:target
+				STUNNER = scope:owner
+			}
+		}
+
+		if = {
+			limit = {
+				exists = scope:scheme_discovered
+			}
+			scope:owner = {
+				save_scope_as = left_portrait
+			}
+		}
+		else = {
+			scope:target = {
+				save_scope_as = left_portrait
+			}
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.5001.a
+		trigger = {
+			exists = scope:scheme_discovered
+		}
+		play_music_cue = "mx_cue_stress"
+		hostile_scheme_opinion_effect = {
+			VICTIM = scope:target
+			ATTACKER = scope:owner
+			SCHEME = stasis_trap_scheme
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.5001.a
+		trigger = {
+			NOT = { exists = scope:scheme_discovered }
+		}
+		play_music_cue = "mx_cue_stress"
+	}
+}
+
+###################################
+# TARGET FAILURE NOTIFICATIONS
+###################################
+
+#I was almost stunned
+stasis_trap_outcome.5002 = {
```

Comment:
I guess this event's left character and its animations can be made like `stasis_trap_outcome.5001`.